lib/output.js

```diff
@@ -1185,6 +1186,35 @@ function OutputStream(options) {
         output.print(self.getValue());
     });
     DEFPRINT(AST_String, function(self, output){
+        var parent = output.parent(1);
+        // Check if there is a need to break directive by inserting an additional semicolon
+        if (parent instanceof AST_Scope) {
+            var print_semicolon = false;
+            // Check if string is the first in body or after directive
+            for (var i = 0; i < parent.body.length; i++) {
+                if (parent.body[i] instanceof AST_SimpleStatement && parent.body[i].body instanceof AST_String) {
+                    print_semicolon = parent.body[i].body === self;
+                    break;
+                }
+                if (!(parent.body[i] instanceof AST_Directive)) {
+                    break; // Breaking directive prologue
+                }
+            }
+            if (print_semicolon) {
+                var str = output.encode_string(self.getValue(), self.quote);
+                // Only output semicolons if string might be a valid directive
+                if (str.indexOf("\\") === -1) {
+                    var last = output.last();
+                    // Extra semicolon might be needed to break directive prologue
+                    if (last !== null && /[;{]\s*$/.test(last)) {
+                        output.force_semicolon();
+                    }
+                    output.force_semicolon();
+                }
+                output.print(str);
+                return;
+            }
+        }
```

Comment:
It has to detect if there is a semicolon in front or if there aren't 2 semicolons before (to add 1 or 2 semicolons in the output) and has to check if the semicolons are needed in case the string itself is an invalid directive anyway.